### ISSUE TITLE
R410a - Glide do not match with HVAC litterature and tools

### ISSUE THREAD

**Issue by charlesp-l** [usnistgov/REFPROP-issues#402]
### Description

Hello to you,

I'm having some issues with the calculation of the R410a dew and bubble points. I want to produce tables that list the dew and bubble point for R410a around 110°F (314.5 to 316.5K, 2.58 to 2.72 MPa). In that range, the glide (difference between dew and bubble temperatures at a constant pressure) of R410a is small compared to other refrigerants, but, I still expected it to be between 0.1 and 0.2°K. Using two other sources of data (linked below), we see that saturation temperature are a little bit different, still, the glide shows an order of magnitude of difference between REFPROP and the other methods.

Here are some results:
Pressure  | REFPROP Dew| REFPROP Bubble| Glide |Danfoss Dew | Danfoss Bubble| Glide | Chemours Dew | Chemours Bubble | Glide|
--------- |:---------------:|:----------------:|:------:|:-------------:|:---------------:|:------:|:----------------:|:----------:|:--:
[MPa(a)] | [°C]       |  [°C]       |    [°K]    |  [°C]   |    [°C]  |   [°K]   |   [°C]   |    [°C]  | [°K]
2.58        | 41.210 |   41.194  | 0.016   | 42.68  |42.56 |   0.16 |  42.82 | 42.68 | 0.14
2.60        | 41.535 |   41.518  | 0.017   | 43.00 | 42.89 | 0.11   |  43.14 | 43.00 | 0.14
2.70        | 43.132 |   43.114  | 0.018   |44.59  | 44.47 |  0.13  |  44.73 | 44.60 | 0.13
 
Danfoss numbers are calculated using the [Danfoss Refrigerant Slider](https://reftools.danfoss.com/spa/tools/ref-slider) 
Chemours numbers are calculated using the [thermodynamic properties of R410a](https://www.freon.com/en/-/media/files/freon/freon-410a-si-thermodynamic-properties.pdf), using Excel Solver to get the temperature as the supplied equations give the pressure as a function of the temperature. Dupont also published [these same values](http://pawel_obstawski.users.sggw.pl/pliki/R410A.pdf).

So, to be honest, I'm not sure of what exactly should be considered as the right values, or what is the accuracy (tolerances) of any of these values. Any clues as to what might be right or not would be appreciated!

A quick look at other points does reveal the same trend:

| Pressure | REFPROP Dew | REFPROP Bubble | Glide | Danfoss Dew | Danfoss Bubble | Glide | Chemours Dew | Chemours Bubble | Glide |
|----------|-------------|----------------|-------|-------------|----------------|-------|--------------|-----------------|-------|
| [MPa(a)] | [°C]        | [°C]           | [°K]  | [°C]        | [°C]           | [°K]  | [°C]         | [°C]            | [°K]  |
| 0.40      | -21.749     | -21.754        | 0.005 | -19.95      | -20.05         | 0.10  | -19.97       | -20.04          | 0.07  |
| 0.80      | -1.622      | -1.622         | 0     | 0.08        | -0.03          | 0.11  | 0.14         | 0.04            | 0.1   |
| 1.20      | 11.831      | 11.827         | 0.004 | 13.46       | 13.35          | 0.11  | 13.55        | 13.44           | 0.11  |
| 1.60      | 22.229      | 22.222         | 0.007 | 23.80       | 23.69          | 0.11  | 23.91        | 23.79           | 0.12  |
| 2.0       | 30.821      | 30.808         | 0.013 | 32.35       | 32.23          | 0.12  | 32.46        | 32.33           | 0.13  |

### Versions

**REFPROP Version:** REFPROP 10
**Operating System and Version:** Windows 10 Pro, 1909
**Access Method:** GUI  

### Additional Information

Tables in REFPROP:
![image](https://user-images.githubusercontent.com/22650753/115400901-ca4c0980-a1b7-11eb-8ff1-ebd597883059.png)

Danfoss Refrigerant slider setup:
![image](https://user-images.githubusercontent.com/22650753/115300864-7b0dc680-a12e-11eb-9884-6ffbdbbda77f.png)

Excel setup for the calculation of the saturated points using Chemours data:
![image](https://user-images.githubusercontent.com/22650753/115299362-b1e2dd00-a12c-11eb-9404-b87b7c2ce779.png)

**Comment by ianhbell** [comment]
Not only do the glides not match, the values don't either.  There must be something messed up on your side, since I did the same calculation in Python, and got the correct values with REFPROP 10.0.0:
```Python
from ctREFPROP.ctREFPROP import REFPROPFunctionLibrary

root = 'C:/Program Files (x86)/REFPROP'
RP = REFPROPFunctionLibrary(root)

for p in [2.58e6, 2.6e6, 2.70e6]:
    for fluid in ['R410A.PPF', 'R410A.MIX']:
        TL = RP.REFPROPdll(fluid, "PQ", "T", RP.MOLAR_BASE_SI, 0,0, p,0,[]).Output[0]
        TV = RP.REFPROPdll(fluid, "PQ", "T", RP.MOLAR_BASE_SI, 0,0, p,1,[]).Output[0]
        print(fluid, TL-TV, TL-273.15, TV-273.15)
```
yielding
```
R410A.PPF -0.11924137306516513 42.56196381163522 42.68120518470039
R410A.MIX -0.11684877260194071 42.563650044873214 42.680498817475154
R410A.PPF -0.11915434844422634 42.885281221764785 43.00443557020901
R410A.MIX -0.11664279572670466 42.88719799405135 43.00384078977805
R410A.PPF -0.11859708265239988 44.47390266187472 44.59249974452712
R410A.MIX -0.11550143245534628 44.476993981778094 44.59249541423344
```

**Comment by ianhbell** [comment]
I tried the same calculation in the GUI, got the right answer: 
![image](https://user-images.githubusercontent.com/11966765/115407675-3b8ebb00-a1be-11eb-9d1c-f12c5fcf1980.png)

How did you generate that table (in detail)?

**Comment by charlesp-l** [comment]
Ian,

Here is how I get to theses results:

1- I selected the R410a mixture:
![image](https://user-images.githubusercontent.com/22650753/115407814-5d883d80-a1be-11eb-9cc2-d42702fb84c1.png)

2 - Select the Calculate -> Saturation Table:
![image](https://user-images.githubusercontent.com/22650753/115408871-38e09580-a1bf-11eb-9b79-553ab58d1fcd.png)

3- Select "Liquid at Bubble Point", then select Vary Pressure: (I would choose "Vapor at dew point" to get the dew point)
![image](https://user-images.githubusercontent.com/22650753/115408959-51e94680-a1bf-11eb-814d-abdc3c4c862a.png)

4 - Enter the parameters of the Pressure range:
![image](https://user-images.githubusercontent.com/22650753/115409080-6b8a8e00-a1bf-11eb-8ab2-04f474ce7a6d.png)

5- Witness the results:
![image](https://user-images.githubusercontent.com/22650753/115409330-a4c2fe00-a1bf-11eb-9055-86ebf8dd2426.png)

The R410A.MIX file (in the MIXTURES directory) dates to 2018-06-04 and has a SHA-256 hash of A76FC73AECA16FB5D3510AF8635C64EE190308093C4F372466D42527E598C5CE

The REFPROP DLL has this hash : FF17E8E9183AB25A10EE3D59954BCB207FACE3B62D549F915160F1FE8C5BA652

**Comment by ianhbell** [comment]
This must be related to miniREFPROP.  I did what you said on my side with REFPROP 10, and it works fine:

![image](https://user-images.githubusercontent.com/11966765/115441480-e616d600-a1de-11eb-804b-fca05ca4673b.png)

**Comment by EricLemmon** [comment]
There's an updated version of mini-Refprop available, try this one and see if the problems are gone:

https://trc.nist.gov/refprop/MINIREF/MINIREF.HTM

**Comment by charlesp-l** [comment]
Eric, I have installed the new version, yet the results are the same. 

I don't know if this helps, but I check on another computer I had laying around that had an old installation of Mini-Refprop (installed around Sept 1st, 2020, ?compilation date 2020-05-14?). This installation gave me the results Ian posted. On that same computer, I installed the new version (in another directory), copy the R32.fld, R125.fld and R410A.mix files from the old to the new installation, and now I get the same (wrong) results that I posted on the top...
![image](https://user-images.githubusercontent.com/22650753/116290491-80d45f00-a761-11eb-8d5c-f353b4ef71e2.png)
